### DISCUSSION TITLE
How can I change layout theme

### DISCUSSION THREAD

**Discussion by chenxiaofie** [vueComponent/ant-design-vue#6471] [Q&A] [answered]
I want to change some input field value, but i not find this

**Comment by shadowgxq** [comment, marked as answer]
sir, see here
you can config in your webpack config:
// webpack.config.js
module.exports = {
  rules: [{
    test: /\.less$/,
    use: [{
      loader: 'style-loader',
    }, {
      loader: 'css-loader', // translates CSS into CommonJS
    }, {
      loader: 'less-loader', // compiles Less to CSS
+     options: {
+       lessOptions: {
+         modifyVars: {
+           'primary-color': '#1DA57A',
+           'link-color': '#1DA57A',
+           'border-radius-base': '2px',
+         },
+         javascriptEnabled: true,
+       }
+     },
    }],
    // ...other rules
  }],
  // ...other config
}
